src/ophys_etl/modules/segmentation/modules/roi_merging.py

```diff
@@ -0,0 +1,88 @@
+import argschema
+import pathlib
+import h5py
+import json
+
+from ophys_etl.modules.segmentation.modules.schemas import \
+    RoiMergerSchema
+
+import ophys_etl.modules.segmentation.postprocess_utils.roi_merging as merging
+from ophys_etl.modules.segmentation.graph_utils.plotting import graph_to_img
```

Comment:
this is now in `ophys_etl.modules.segmentation.graph_utils.conversion`
you may encounter some merge conflicts, though it doesn't seem to be complaining.

Reply:
I wonder if my environment has gotten too polluted to realize this... anyway: I've fixed it.